l3kernel/testfiles/m3text002.ptex.tlg

```diff
@@ -390,27 +424,27 @@ l. ...  }
 l. ...  }
 ============================================================
 ============================================================
-TEST 31: Case change switching
+TEST 32: Case change switching
 ============================================================
 lower
 UPPER
 Title
 Title
 WORDS lower
 words UPPER
-Words UPPER
-Words \text_case_switch:nnnn {normal}{lower}{UPPER}{Title}
+Words Title
+Words Title
```

Comment:
Is there going to be a test case for `\text_sentencecase` here as well?

Reply:
That wasn't the plan. Lower-, upper- and titlecasing have specific Unicode characters involved, so are 'fundamentally distinct'. Sentence casing isn't the same: you are applying titlecasing in a selective fashion. The switching mech. is there to handle the situation where something non-expandable needs to change depending on lower/upper/title status, so really shouldn't need to be extended as far as I can see (counter-examples of course could show I'm wrong). Any extension would need a new function, and that would be tricky. at the 2e level.

Reply:
But `\text_sentencecase` *does* interface with `\text_case_switch:nnnn`, right? At the start we get the "Title" output and afterwards "lower", or is it different? (I guess I just wanted to make sure this is the case and since I didn't manage to run the code I just wanted to see a test here that confirmed this.)

Reply:
Yes, perhaps I should make that clearer: sentence casing means applying titlecasing only to the first word of the input. (I may eventually look at dealing with multiple sentences, which would mean of course it would apply titlecasing to the start of each sentence: looks quite tricky/slow.)